components/butterfly/src/server/inbound.rs

```diff
@@ -109,8 +109,8 @@ impl<'a> Inbound<'a> {
                 }
                 Err(e) => {
                     match e.raw_os_error() {
-                        Some(35) => {
-                            // This is the normal non-blocking result
+                        Some(35) | Some(11) => {
```

Comment:
So, as @reset points out, they are super duper different. The names are POSIX standards; the numbers, well, those can be whatever. :)